### PR TITLE
Add initial work on specification file

### PR DESCRIPTION
**Pull request by Eeems**

### CODE REVIEW

File: Specification.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+Application Registration Specification
+============================================
+
+Application registration files are used to indicate that an application is installed on the system. This then allows third party launchers and applications to be aware of how to launch other applications. It's meant to facilitate package managers adding and removing applications and automatically reloading third party launchers. These files must have a file extension of `remarkable` and must be located in `/opt/usr/share/applications`.
```
Comment (matteodelabre) [inline Specification.md:4]: `rmapp` seemed to have the most support during our meeting.

```suggestion
Application registration files are used to indicate that an application is installed on the system. This then allows third party launchers and applications to be aware of how to launch other applications. It's meant to facilitate package managers adding and removing applications and automatically reloading third party launchers. These files must have a file extension of `rmapp` and must be located in `/opt/usr/share/applications`.
```

File: Specification.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+permissions: [power, wifi]
+events:
+  resume: touch /tmp/.sample-running
+  pause: rm /tmp/.sample-running
+```
+
+
+
+Oxide Properties Documentation
+----------
```
Comment (matteodelabre) [inline Specification.md:86]: I think the specification should not define any Oxide-specific property.

Comment (Eeems) [reply Specification.md:86]: For sure, I was leaving it in as context. That's why this is still marked as a draft pull. More for context on the first meeting that we can start working though.

**Comment by Eeems** [comment]
We've decided to go with a subset of XDG.

**Pull request closed without merging**